frontend/src/App.js

```diff
@@ -12,7 +12,6 @@ class App extends Component {
 
   // Test to see if backend servlet connects to frontend.
   componentDidMount() {
-    console.log("Inside componentDidMount!");
     fetch("/api/v1/test-servlet")
```

Comment:
nit: due to the way the component lifecycle is done for react, for future PRs prefer to not have the console.log statement but instead a test maybe if the ftech works, then that message is displayed on the screen, or if the fetch is successful, then the <Home> will be rendered, or something like that.

this would give better insight into how the `componentDidMount` and `fetch` call relates to the actual UI (the `render` part).

Reply:
good point, thanks! will keep in mind for future PRs